src/server/services/user/editUser.js

```diff
@@ -22,6 +23,12 @@ const { getConnection, dropConnection } = require('../../db');
 		terminateReadline('Password must be at least eight characters, user\'s password not modified');
 	}
 
+	role = await ask('Role: '); // untested
```

Comment:
It's untested because I didn't formally check that this works (by running it or writing a mocha test). I am fairly confident that it should work as intended though. Do you think I should add a mocha test?

Reply:
I think this is fine but the comment confused me.